k9mail/src/main/java/com/fsck/k9/ui/crypto/MessageCryptoHelper.java

```diff
@@ -344,18 +340,29 @@ public void onProgress(int current, int max) {
         });
     }
 
+    /**
+     * Common operations preceding {@link #onCryptoOperationReturned(MimeBodyPart)}
+     * @param result
+     * @param decryptedPart
+     */
+    private void onCryptoOpReturnedPreops(Intent result, MimeBodyPart decryptedPart) {
```

Comment:
`onCryptoOperationReturned` is never called otherwise, might as well move the contents of this method in there